### ISSUE TITLE
Cannot read property 'Symbol(Symbol.iterator)' of null error on CSS change

### ISSUE THREAD

**Issue by JayAndCatchFire** [parcel-bundler/parcel#811]
**🐛 bug report**

This may be related to https://github.com/parcel-bundler/parcel/issues/344 as a similar error message is reported in that ticket.

### 🎛 Configuration (.babelrc, package.json, cli command)
I have no `.babelrc`.

#### package.json
```
{
  "name": "parcel-playground",
  "version": "0.1.0",
  "devDependencies": {
    "parcel-bundler": "^1.5.1"
  }
}
```

#### cli command
```bash
yarn parcel ./index.html
```

### 🤔 Expected Behavior

I didn't expect any error messages.

### 😯 Current Behavior

When I change my CSS file and save it, I receive errors in the terminal and the devtools console.  It seems to only be intermittently repeatable.  It can happen when I save the file without actually changing anything.  If I spam <kbd>Ctrl</kbd> + <kbd>S</kbd> in my editor it randomly occurs.

#### terminal

```bash
🚨  Cannot read property 'Symbol(Symbol.iterator)' of null
    at Bundler.bundle (/home/me/code/parcel-playground/node_modules/parcel-bundler/src/Bundler.js:179:61)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

#### devtools console

```
css-loader.js:9 [parcel] 🚨  loadedAssets is not iterable
data.error.stack
ws.onmessage @ css-loader.js:9
css-loader.js:9 [parcel] 🚨  Cannot read property 'Symbol(Symbol.iterator)' of null
data.error.stack
```

### 🔦 Context

I was just trying out parcel for the first time.

### 💻 Code Sample

These are literally the files I was using. I based them on the sample code at https://parceljs.org/.

#### index.html
```html
<html>
<body>
  <script src="./index.js"></script>
</body>
</html>
```

#### main.css
```css
.main {
  /* This file was copied from http://placekitten.com/200/200?image=10 */
  background: url('./200.jpeg');
  color: red;
}
```

#### index.js
```js
import main from './main';

main();
```

#### main.js
```js
import classes from './main.css';

export default () => {
  console.log(classes.main);
};
```

### 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel | 1.5.1 |
| Node | 8.9.4 |
| Yarn | 1.3.2 |
| Operating System | Ubuntu 16.04 |

**Comment by DeMoorJasper** [comment]
This should be fixed in current master and upcoming 1.6

**Comment by JayAndCatchFire** [comment]
I just upgraded to 1.6.1 and I can confirm that this is no longer happening.
